packages/projection/src/operators/Mappers/withAddresses.ts

```diff
@@ -3,6 +3,7 @@ import { Hash28ByteBase16 } from '@cardano-sdk/crypto';
 import { WithUtxo } from './withUtxo';
 import { unifiedProjectorOperator } from '../utils';
 import uniq from 'lodash/uniq.js';
+import { credentialsFromAddress } from '@cardano-sdk/core/src/Cardano/util';
```

Comment:
This is going to cause trouble; should import from index, e.g. `import { Cardano } from '@cardano-sdk/core'` then use it like `Cardano.util.credentialsFromAddress`